sys/include/stdio_nimble.h

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_stdio_nimble STDIO over UART + NimBLE
+ * @ingroup     sys
+ *
+ * @brief       Standard input/output backend using UART and NimBLE.
+ *              Define STDIO_NIMBLE_DISABLE_UART in CFLAGS to disable the UART part.
+ *
+ * @{
+ * @file
+ *
+ * @author      Hendrik van Essen <hendrik.ve@fu-berlin.de>
+ */
+
+#ifndef STDIO_NIMBLE_H
+#define STDIO_NIMBLE_H
+
+#include "stdio_base.h"
+
+#include "nimble_riot.h"
+#include "net/bluetil/ad.h"
+
+#include "host/ble_hs.h"
+#include "host/util/util.h"
+#include "host/ble_gatt.h"
+#include "services/gap/ble_svc_gap.h"
+#include "services/gatt/ble_svc_gatt.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Nimble handle for stdout characteristic
+ */
+static uint16_t _val_handle_stdout;
+
+/**
+ * @brief UUID for stdio service (value: e6d54866-0292-4779-b8f8-c52bbec91e71)
+ */
+static const ble_uuid128_t gatt_svr_svc_stdio_uuid
+        = BLE_UUID128_INIT(0x71, 0x1e, 0xc9, 0xbe, 0x2b, 0xc5, 0xf8, 0xb8, 0x79,
+                           0x47, 0x92, 0x02, 0x66, 0x48, 0xd5, 0xe6);
+
+/**
+ * @brief UUID for stdout characteristic (value: 35f28386-3070-4f3b-ba38-27507e991762)
+ */
+static const ble_uuid128_t gatt_svr_chr_stdout_uuid
+        = BLE_UUID128_INIT(0x62, 0x17, 0x99, 0x7e, 0x50, 0x27, 0x38, 0xba, 0x3b,
+                           0x4f, 0x70, 0x30, 0x86, 0x83, 0xf2, 0x35);
+
+/**
+ * @brief UUID for stdin characteristic (value: ccdd113f-40d5-4d68-86ac-a728dd82f4aa)
+ */
+static const ble_uuid128_t gatt_svr_chr_stdin_uuid
+        = BLE_UUID128_INIT(0xaa, 0xf4, 0x82, 0xdd, 0x28, 0xa7, 0xac, 0x86, 0x68,
+                           0x4d, 0xd5, 0x40, 0x3f, 0x11, 0xdd, 0xcc);
+
+/**
+ * @brief Dummy access callback, because nimble requires one
+ */
+static int gatt_svr_chr_access_noop(
```

Comment:
if this function is implemented in the header file, it should at least be `inlined`. But why not put it into the `*.c` file?!

Reply:
Moved in to .c